src/lib_ccx/ocr.c

```diff
@@ -659,6 +660,8 @@ char *ocr_bitmap(void *arg, png_color *palette, png_byte *alpha, unsigned char *
 						last_font_tag = font_tag;
 					}
 					last_font_tag_end = strstr(last_font_tag, ">");
+					if (last_font_tag_end > line_end)
+						last_font_tag_end = NULL;
```

Comment:
Let's put this line in its own block ( { } )
I know most of our code doesn't do it but let's say single line branches without blocks are out of fashion these days to how prone they are to introduce bugs and security problems :-)

Reply:
I'll try and make the changes and commit withnin the day :)

Reply:
I'll try and make the changes within the day :)